src/sortStudents.ts

```diff
@@ -1,16 +1,60 @@
 
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades'
 }
 
-// create SortOrder type
-export type SortOrder;
+function gradesCalc(arr: number[]): number {
+  return arr.reduce((valueA, valueB) => valueA + valueB) / arr.length;
+}
+
+export type SortOrder = 'asc' | 'desc';
```

Comment:
Better describe all interfaces, enums, types and after your functions 

```suggestion
export type SortOrder = 'asc' | 'desc';

function gradesCalc(arr: number[]): number {
  return arr.reduce((valueA, valueB) => valueA + valueB) / arr.length;
}
```